recipe/meta.yaml

```
@@ -38,18 +32,18 @@ outputs:
         - pip
         - setuptools
         - wheel
```

https://github.com/matplotlib/basemap/blob/v2.0.0/pyproject.toml#L4
```suggestion
        - wheel >=0.40,< 0.46
```